### PR TITLE
Adding framerate control for Vimba v6

### PR DESCRIPTION
**Pull request by MikoMikarro**

### CODE REVIEW

**Comment by icolwell-as** [comment]
Thanks @MikoMikarro for the PR! Can you provide some more context and details in the PR description?
For example:
- Why is this new parameter needed?
- What problem did you run into that this is solving?
- What camera hardware are you using?

**Comment by MikoMikarro** [comment]
Hi, we added the new name for the frame parameter that was added with the last firmware, this fixes some of the problems related to issue #86 .  Without this change in the parameter name, we couldn't change the camera's frame rate. I've also left uncommented the last name so it should work without any problem in the previous and current firmware.

I've also added the parameter that allows to use free run or fix the framerate.

Our team is using the G-201 camera: https://www.alliedvision.com/en/camera-selector/detail/manta/g-201/

File: cfg/AvtVimbaCamera.cfg [hunk 1]
```diff
@@ -160,6 +160,7 @@ exposure_mode_enum = gen.enum([gen.const("Timed", str_t, "Timed", "[Default] Cam
 # ACQUISITION
 gen.add("acquisition_mode",       str_t,    SensorLevels.RECONFIGURE_STOP,    "Camera acquisition mode", "Continuous", edit_method = acquisition_mode_enum)
 gen.add("acquisition_rate",       double_t, SensorLevels.RECONFIGURE_RUNNING, "Sets the expected triggering rate in externally triggered mode.", 10, 1, 35)
+gen.add("acquisition_rate_enable",       bool_t, SensorLevels.RECONFIGURE_RUNNING, "Allows to fix acquisition rate, if available", False)
```
Comment (icolwell-as) [inline cfg/AvtVimbaCamera.cfg]: Can you adjust the whitespace formatting here so that bool_t aligns vertically with all the other field types?

**Review by icolwell-as — CHANGES_REQUESTED** [review]
Thanks for the details! these change make sense to me, just a small nitpick on the formatting.

Thanks again for taking the time to submit a PR!

**Review by icolwell-as — DISMISSED** [review]

File: launch/Mako_G-319.launch [hunk 2]
```diff
@@ -13,6 +14,7 @@
     <arg name="guid" value="$(arg guid)"/>
     <arg name="camera_info_url" value="$(arg camera_info_url)"/>
     <arg name="acquisition_rate" value="$(arg acquisition_rate)"/>
+    <arg name="acquisition_rate_enable" default="$(arg acquisition_rate_enable)"/>
```
Comment (shan-as) [inline launch/Mako_G-319.launch]: `value` instead of `default` here ?

**Review by shan-as — APPROVED** [review]
Have not tested on a camera

**Review by dvangeyn-as — APPROVED** [review]

**Pull request merged**